buildSrc/build.gradle.kts

```diff
@@ -1,6 +1,6 @@
 plugins {
     `kotlin-dsl`
-    kotlin("plugin.serialization") version "1.9.22"
+    kotlin("plugin.serialization") version "2.0.20"
```

Comment:
Just used the version that matched in the warning message when I set this to `2.1.20`